src/csvcubed/models/cube/qb/components/concept.py

```diff
@@ -38,22 +47,71 @@ def __eq__(self, other):
     def __hash__(self):
         return self.code.__hash__()
 
+    def validate_data(
```

Comment:
Again, this shouldn't be in here. Make sure to tidy up the imports after you remove it.

Reply:
Removed it